packages/backend/src/ee/database/migrations/20240522101742_add_embedding_vector_to_catalog.ts

```diff
@@ -15,5 +15,9 @@ export async function down(knex: Knex): Promise<void> {
         `ALTER TABLE ${CatalogTableName} DROP COLUMN IF EXISTS embedding_vector`,
     );
 
-    await knex.raw('DROP EXTENSION IF EXISTS vector');
+    await knex.raw('DROP EXTENSION IF EXISTS vector').catch(() => {
```

Comment:
Catch error when database user doesn't have permissions to drop extension.
<img width="971" alt="Screenshot 2025-01-29 at 16 14 20" src="https://github.com/user-attachments/assets/3b4bd53b-97cb-4cbc-a27a-02a9f9fc1f10" />

Reply:
cc: @IrakliJani